src/schema/v2/SearchableItem/SearchableItemPresenter.ts

```diff
@@ -57,6 +58,8 @@ export class SearchableItemPresenter {
       case "Booth":
       case "PartnerShow":
         return `/show/${id}`
+      case "ArtistSeries":
+        return `/artist-series/${id}`
```

Comment:
is `id` the slug in this case?

Reply:
It always is for search results (with a fallback to database id).

Coming from https://github.com/artsy/gravity/blob/f52e989956c6b4f8703db6f6b04d3d1fbe58fd11/app/api/util/search_helpers.rb#L48-L52